src/TheRestOfYourLife/quad.h

```diff
@@ -26,6 +26,8 @@ class quad : public hittable {
         D = -dot(normal, corner);
         w = n / dot(n,n);
 
+        area = n.length();
+
```

Comment:
I don't see these changes in the text?

Same with area below.

Reply:
Wow, good catch.